public/store/app/index.ts

```diff
@@ -9,13 +10,20 @@ export interface AppState {
   statusPanelState: StatusPanelState;
 }
 
+export interface SatelliteBand {
```

Comment:
From what I read, the band are not defined by a mix of RGB colours, but by bands number.
Or is the goal of the satellite band primitive to make false colours by transforming up to 3 bands into RGB?